### DISCUSSION TITLE
Is there need a http reader or http sender in source connector?

### DISCUSSION THREAD

**Discussion by linxd** [nextgenhealthcare/connect#5934] [Feature/Enhancement Ideas]
we open want sending a http request to a http service , and then process the response in the next step. 
now , I just do by adding null source js reader ,and in destination, first add a http sender to post the http request. and then process in the next destination.

is there  some good way?

**Comment by pacmano1** [comment]
Yes, that works.  You can also process the response in the response transformer. 

As an alternative, you can make java http requests on the source side in a JS reader or transformer.
